hw/formal/formal_conn.sh

```diff
@@ -26,7 +26,7 @@ set -e
 CORE_PATH=systems:top_earlgrey
 REPO_PATH=$(readlink -f ../..)
 
-export TOP="top_earlgrey"
+export DUT_TOP="top_earlgrey"
```

Comment:
can we get this from the dvsim flow?

Reply:
yes, dvsim flow won't use `formal_conn.sh` approach. `formal_conn.sh` is a separate script to temp run with GUI (dvsim formal does not support GUI mode so far). Sorry for the confusion.

Reply:
I see. Thanks for the explanation.

Reply:
Actually DVSim does support GUI now (needs a minor fix for formal).